src/test/java/racingcar/RandomMakerTest.java

```diff
@@ -14,4 +15,4 @@ void random() {
 
         assertThat(actual).isGreaterThan(-1).isLessThan(10);
```

Comment:
```suggestion
        assertThat(actual).isNotNegative().isLessThan(10);
```
`isNotNegative()` 를 사용할 수도 있답니당

Reply:
다양한 함수들을 더 찾아봐야겠네요 감사합니다 🙇‍♂️